tests/web-server/app/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
We should consider using npm workspaces on the test level for easier maintenance.

Reply:
I haven't used them in more than 3 years, so I need to refresh my knowledge.
Can you please let me know what value we see from using them?

Undoubtedly, this will cause me to slow down. This can't be the excuse to sacrifice quality, but due to the limited time I have, I could use help from you, or others, to implement this.

Reply:
The main benefit is having a central place to overview and manage the test apps. As the number of test apps grow, so will the value of this approach.
For the time being, we can leave this change out of this PR, and prioritize it as a part of future changes.